analyzers/src/SonarAnalyzer.CSharp/Rules/CallToAsyncMethodShouldNotBeBlocking.cs

```diff
@@ -33,60 +33,54 @@ namespace SonarAnalyzer.Rules.CSharp
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public sealed class CallToAsyncMethodShouldNotBeBlocking : SonarDiagnosticAnalyzer
     {
-        private const string DiagnosticId = "S4462";
         private const string MessageFormat = "Replace this use of '{0}' with '{1}'.";
         private const string ResultName = "Result";
         private const string ContinueWithName = "ContinueWith";
         private const string SleepName = "Sleep";
+        private const string AzureFunctionSuffix = @" Calls to ""async"" methods should not be blocking in Azure Functions.";
```

Comment:
This is not precise and does not fit in cases like "WaitAny". You don't await a method result, but a Task (or even more precise "an awaitable object"). Maybe just 
```suggestion
        private const string AzureFunctionSuffix = @" Don't perform blocking operations in Azure Functions.";
```